plugins/org.wso2.developerstudio.eclipse.artifact.messageprocessor/src/org/wso2/developerstudio/eclipse/artifact/messageprocessor/wizard/MessageProcessorCreationWizard.java

```diff
@@ -229,6 +229,10 @@ private String getTemplateContent() {
 				messageProcessorPrameeters.put("max.delivery.attempts",
 						messageProcessorModel.getDeliveryAttempts());
 			}
+			
+			messageProcessorPrameeters.put("max.store.connection.attempts", Integer.toString(messageProcessorModel.getMaxStoreConnectionAttempts()));
```

Comment:
Formatting issue(line length) shall we fix this?

Reply:
Need to keep it as it is as to fix this need to reformat the whole code.